specs/wp-post-nux-spec.js

```diff
@@ -59,13 +61,50 @@ test.describe( `[${host}] Post-NUX Flows (${screenSize}) @parallel`, function()
 				} );
 
 				test.it( 'Can update and view site title', function() {
-					const newSiteTitle = dataHelper.randomPhrase();
 					this.customizerPage.setTitle( newSiteTitle );
 					return this.customizerPage.previewTitle().then( ( titleShown ) => {
 						assert.equal( titleShown.toUpperCase(), newSiteTitle.toUpperCase(), 'The customizer preview title shown does not reflect the title input' );
 					} );
 				} );
 
+				test.it( 'Can open publish settings', function() {
+					return this.customizerPage.openOrClosePublishSettings();
+				} );
+
+				test.it( 'Can select "Save Draft" action in publish settings', function() {
+					return this.customizerPage.selectActionSaveDraft();
+				} );
+
+				test.it( 'Can save the theme', function() {
+					return this.customizerPage.saveNewTheme();
+				} );
+
+				test.it( 'Can navigate to preview URL', function() {
+					return this.customizerPage.openPreviewURL();
+				} );
+
+				test.it( 'Can see the correct title displayed', function() {
+					switchToWindow( driver, 1 );
+					return this.customizerPage.title().then( ( title ) => {
+						return assert( title.includes( newSiteTitle ), 'The title is correctly displayed on the preview' );
+					} );
+				} );
+
+				test.it( 'Can revert the site title', function() {
+					switchToWindow( driver, 0 );
```

Comment:
This leaves the new window open which isn't ideal, especially if you had a subsequent test that opens a new window